.github/workflows/ci.yml

```diff
@@ -34,8 +34,11 @@ jobs:
 
           - operating-system: 'ubuntu-20.04'
             php-version: '8.0'
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.1'
             job-description: 'with Symfony ^6'
-            execute-flex-with-symfony-version: '^6' # explicit check for Symfony 6.x compatibility
+            execute-flex-with-symfony-version: '^6.1' # explicit check for Symfony 6.x compatibility
```

Comment:
why the change?

Reply:
It is probably leftover from initial ParaTest (in general fix dependency resolution), I can try running pipeline without this change to see how it works with ParaUnit.

Reply:
It was reverted. I believe we should change it anyway, because now we don't test with latest Symfony, but only on 6.0. But this is out of scope of this PR 🙂.

Reply:
why `^6` tests only with `6.0`?

Reply:
@keradus because [Symfony 6.1 requires PHP 8.1](https://symfony.com/blog/symfony-6-1-will-require-php-8-1) and we have `execute-flex-with-symfony-version: ^6` on PHP 8.0 in CI matrix. That's why this change was done before 😉.